docs/CHANGES.md

```diff
@@ -0,0 +1 @@
+- Add LibreJS support
```

Comment:
This should be added to the CHANGES.md file at the root of the repository.

Reply:
Done